File: py/menu.py
print("MENÚ")
print("1. Saludar")
print("2. Fecha")
print("3. Salir")

opcion = int(input("Seleccione una opción: "))

if opcion == 1:
    print("Elegiste Saludar")
    print ("Hola, como estás?")
elif opcion == 2:
    print("Elegiste Fecha")
    print("Hoy es 23 de Mayo del 2015")
elif opcion == 3:
    print("Elegiste Salir")
    print("saliendo del programa...")
else:
    print("Error")

File: menu.js
console.log("MENÚ");
console.log("1. Saludar");
console.log("2. Fecha");
console.log("3. Salir");

let opcion = 3

if (opcion === 1) {
    console.log("Elegiste Saludar")
    console.log("Hola, cómo estás?");
} else if (opcion === 2) {
    console.log("Elegiste Fecha")
    console.log("Estamos a 12 de enero del 1099");
} else if (opcion === 3) {
    console.log("Elegiste Salir")
    console.log("Saliendo del programa...");
} else {
    console.log ("Error")
}